### ISSUE TITLE
Unsupported Block Editor: Toolbar is not displayed in self-hosted sites

### ISSUE THREAD

**Issue by fluiddot** [wordpress-mobile/gutenberg-mobile#5721]
**Describe the bug**
The toolbar is not displayed when editing an unsupported block via UBE (Unsupported Block Editor) in a self-hosted site. This reduces the functionality of editing the block in UBE.

**To Reproduce**
1. Follow the steps from ["Editing unsupported blocks is enabled on self-hosted sites accessed via Jetpack"](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#editing-unsupported-blocks-is-enabled-on-self-hosted-sites-accessed-via-jetpack) to enable UBE in a self-hosted site.
2. Edit an unsupported block in a self-hosted site.
3. When UBE is loaded, observe that the toolbar is not present.

**Expected behavior**
The toolbar should be displayed as when editing an unsupported block in a WPCOM site.

**Screenshots**
<img src=https://user-images.githubusercontent.com/14905380/236179030-6c0e58c7-fefe-4d41-a9af-17cc60908935.PNG width=300>

**Smartphone (please complete the following information):**
 - Device: iPhone 11
 - OS: iOS 15.4.1
 - Version 22.3.0.1

**Additional context**
N/A

**Comment by dcalhoun** [comment]
@fluiddot in this scenario, has the block been "selected" via tapping it? I believe the toolbar is not displayed unless a block is selected. Also, if I recall correctly, the Table block specifically behaves in the UBE oddly at times.

**Comment by fluiddot** [comment]
> @fluiddot in this scenario, has the block been "selected" via tapping it? I believe the toolbar is not displayed unless a block is selected. Also, if I recall correctly, the Table block specifically behaves in the UBE oddly at times.

Ah, good point. I've just selected it and now I see the toolbar. I think I got misled by the behavior on WPCOM sites where the toolbar is automatically displayed when opening UBE. I confirm this is not an actual issue, so I'll close it. Thanks @dcalhoun for the info 🙇 !

<img src=https://user-images.githubusercontent.com/14905380/236227163-53081204-f637-4a5e-b8fe-ae802616d0c2.jpeg width=300>
